packages/bot/src/scratch/hooks/procedures.js

```diff
@@ -68,24 +82,26 @@ Blockly.Procedures.flyoutCategory = function(workspace) {
             const block = document.createElement('block');
             block.setAttribute('type', templateName);
             block.setAttribute('gap', 16);
-
+    
```

Comment:
@sam-binary can you please remove the spaces here

Reply:
I think leave a space there is more neat 😅

Reply:
@sam-binary a space in an empty line!